lib/src/lib.rs

```diff
@@ -42,6 +29,8 @@ pub struct Tester {
     account: Account,
     // The Target Actor to be tested.
     target_actor: Option<DeployedActor>,
+    // The Method message sequence number.
+    sequence: u64,
```

Comment:
Thought we wanted to use [Atomic](https://doc.rust-lang.org/std/sync/atomic/) here. Or did we plan to tackle that when we tackle test concurrency?